tests/unit/job_runner_handlers/test_pbs.py

```diff
@@ -40,8 +43,28 @@
                 '#PBS -e cylc-run/chop/log/job/1/axe/01/job.err',
                 '#PBS -l walltime=180',
             ],
+            id='it sets basic directives'
```

Comment:
FYI these IDs are used on the CLI e.g:

```
$ pytest path/to/file.py::test_function[parameter]
```

Spaces, funny characters and long ids can make CLI work hard.

Non critical (and I've done this myself) but for future I would suggest naming conventions like these:

```python
id='basic'
id='long-job-name'
id='truncated-job-name'
id='custom-directives'
```